confidant/routes/services.py

```diff
@@ -209,16 +209,18 @@ def get_service(id):
     metadata_only = misc.get_boolean(request.args.get('metadata_only'))
     logged_in_user = authnz.get_logged_in_user()
     action = 'metadata' if metadata_only else 'get'
-    permissions['metadata'] = acl_module_check(
-        resource_type='service',
-        action='metadata',
-        resource_id=id,
-    )
-    permissions['get'] = acl_module_check(
-        resource_type='service',
-        action='get',
-        resource_id=id,
-    )
+    if action == 'metadata':
+        permissions['metadata'] = acl_module_check(
+            resource_type='service',
+            action='metadata',
+            resource_id=id,
+        )
+    elif action == 'get':
```

Comment:
```suggestion
    else:
```

Reply:
there's a 'update' in the permission map, so i keep it more strict

Reply:
```
  permissions = {
        'metadata': False,
        'get': False,
        'update': False,
    }
```

Reply:
action can't take any other values besides "get" and "metadata" tho. np. just a nit tho